.github/workflows/release-nightly.yml

```diff
@@ -11,28 +12,47 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-24.04, macos-13, macos-14, windows-2019]
-        build: [cp310, cp311, cp312]
+        os: [ubuntu-24.04]
+        build: [cp312]
     uses: ./.github/workflows/wheel.yml
     with:
       os: ${{ matrix.os }}
       build: ${{ matrix.build }}
 
   release:
-    name: Make Release on GitHub
+    name: Make Release on GitHub and pypi.anaconda.org
     needs: build_wheels
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
+    defaults:
+      run:
+        shell: bash -l {0} # required for mamba envs
     permissions:
       contents: write
     steps:
       - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.10"
       - uses: actions/download-artifact@v4
         with:
           path: dist
           merge-multiple: true
-      - uses: actions/setup-python@v5
+      - name: List old dev wheels from pypi.anaconda.org
+        run: |
+          curl --silent "https://api.anaconda.org/package/scipp-nightly-wheels/scipp" | jq -r '.releases[] | select(.version | contains("dev")) | .full_name' > dev_wheels.txt
+          cat dev_wheels.txt
+      - name: Upload wheel to pypi.anaconda.org/scipp-nightly-wheels/simple
+        uses: scientific-python/upload-nightly-action@3fcd15dcbd3e25176438080493f97592a9c84e0b
         with:
-          python-version: "3.10"
+          artifacts_path: dist
+          anaconda_nightly_upload_organization: scipp-nightly-wheels
+          anaconda_nightly_upload_token: ${{secrets.ANACONDA_NIGHTLY_WHEEL_TOKEN}}
+      - name: Delete old dev wheels from pypi.anaconda.org
+        run: |
+          while read -r package_name; do
+            echo "Removing $package_name"
+            pixi exec -s anaconda-client=1.12.3 anaconda --token "${{secrets.ANACONDA_NIGHTLY_WHEEL_TOKEN}}" remove -f "$package_name"
+          done < dev_wheels.txt
```

Comment:
While the `pixi` setup from the `prefix-dev/setup-pixi` should still be around, it might be safer to try to set it up again as this could potentially get cleaned up in the future.

Nice use of `pixi exec --spec` though! :)

Reply:
Yeah that's probably better, but we may cycle through the removing-wheels-from-anaconda bit later again :)